src/main.ts

```diff
@@ -223,11 +230,26 @@ const command: Command = {
 		});
 		let configs: webpack.Configuration[];
 		if (args.mode === 'dev') {
-			configs = [devConfigFactory({ ...rc, ...commandLineArgs, elements })];
+			configs =
```

Comment:
@mwistrand Do we still need to build legacy and evergreen on dev? If not I suspect we can remove the `clean` logic and just have a single config passed.

```ts
[devConfigFactory({ ...rc, ...commandLineArgs, elements })]
```

and the same for the `dist` config

```
[distConfigFactory({ ...rc, ...commandLineArgs, elements })]
```